File: Lab4CrossingWords/Lab4CrossingWords/CanPlace.cs
using System.Drawing;

namespace Lab4CrossingWords
{
    public class CanPlace
    {
        public bool CanTop { get; set; }

        public Point TopStart { get; set; }


        public bool CanLeft { get; set; }

        public Point LeftStart { get; set; }

        public bool Posible
        {
            get { return (CanLeft || CanTop); }
        }
    }
}

File: Lab4CrossingWords/TestL4/CanPlace.cpp
#include "stdafx.h"
#include "CanPlace.h"

CanPlace::CanPlace()
{
	CanTop = false;
	//TopStart = topStart;
	CanLeft = false;
	//LeftStart = leftStart;
}

CanPlace::CanPlace(bool canTop, Point topStart, bool canLeft, Point leftStart)
{
	CanTop = canTop;
	TopStart = topStart;
	CanLeft = canLeft;
	LeftStart = leftStart;
}

bool CanPlace::Posible() const
{
	return (CanLeft || CanTop);
}
